config/initializers/new_framework_defaults.rb

```diff
@@ -1,20 +0,0 @@
-# Be sure to restart your server when you modify this file.
-#
-# This file contains migration options to ease your Rails 5.0 upgrade.
-#
-# Once upgraded flip defaults one by one to migrate to the new default.
-#
-# Read the Rails 5.0 release notes for more info on each option.
-
-# Enable per-form CSRF tokens. Previous versions had false.
-Rails.application.config.action_controller.per_form_csrf_tokens = true
-
-# Enable origin-checking CSRF mitigation. Previous versions had false.
-Rails.application.config.action_controller.forgery_protection_origin_check = false
-
-# Make Ruby 2.4 preserve the timezone of the receiver when calling `to_time`.
-# Previous versions had false.
-ActiveSupport.to_time_preserves_timezone = false
```

Comment:
Those seem to have been set on purpose to something that's not the default. Are you sure they should be removed?

Reply:
I'd move them to application.rb if they're to be kept, and squash this commit.

Reply:
note that they might not break tests, but crash in real life

Reply:
👍